install/connections.csv

```diff
@@ -543,7 +543,8 @@ MSWZ,10,Moikau Station,,,2003-08-28T20:55:00Z,9999-01-01T00:00:00Z
 MSZ,10,Milford Sound,,0,2005-07-19T20:00:00Z,9999-01-01T00:00:00Z
 MSZ,10,Milford Sound,Short Period,,1990-05-28T11:33:00Z,2005-06-15T04:06:00Z
 MSZ,20,Milford Sound,,3,2005-07-19T20:00:00Z,2017-05-23T04:15:00Z
-MSZ,20,Milford Sound,Strong Motion,,2017-05-23T04:15:01Z,9999-01-01T00:00:00Z
+MSZ,20,Milford Sound,Strong Motion,,2017-05-23T04:15:01Z,2023-06-13T23:17:00Z
+MSZ,21,Milford Sound,Strong Motion,,2023-06-14T03:35:00Z,9999-01-01T00:00:00Z
```

Comment:
Ah yes, as a recorder has gone in I think line 547 isn't needed.

Reply:
Line eliminated. anything else I missed here or is this good?